docs/sources/api/_index.md

```diff
@@ -269,8 +269,8 @@ accepts the following query parameters in the URL:
 
 - `query`: The [LogQL]({{< relref "../logql/" >}}) query to perform
 - `limit`: The max number of entries to return. It defaults to `100`. Only applies to query types which produce a stream(log lines) response.
-- `start`: The start time for the query as a nanosecond Unix epoch or another [supported format](#timestamp-formats). Defaults to one hour ago.
-- `end`: The end time for the query as a nanosecond Unix epoch or another [supported format](#timestamp-formats). Defaults to now.
+- `start`: The start time for the query as a nanosecond Unix epoch or another [supported format](#timestamp-formats). Defaults to one hour ago. (This is inclusive, Loki returns results with timestamp greater or equal to this value)
```

Comment:
I'm a bit confused by the "inclusive" part here.  What is being included here?  I would just remove that bit.

```suggestion
- `start`: The start time for the query as a nanosecond Unix epoch or another [supported format](#timestamp-formats). Defaults to one hour ago. Loki returns results with timestamp greater or equal to this value.
```

Reply:
sure, your change makes sense

(to explain the inclusive/exclusive, it describes whether the `start` timestamp itself is included in the "valid" timestamps. in this case, it is included. but, when talking about the `end` timestamp, the `end` timestamp itself is not included in the "valid" timestamps, because we only take timestamps that are lower than `end`, so that one is exclusive.. but anyway, it's not needed, i'm totally fine with removing it)